Makefile.devcontainer

```diff
@@ -2,22 +2,30 @@
 
 include Makefile.include
 
-release-dev:
+release-dev-managed:
```

Comment:
Can you please also add comments to all these new targets. We are using them to print output of `make help`